### DISCUSSION TITLE
Is the maintenance of this repository alive?

### DISCUSSION THREAD

**Discussion by marisst** [chartjs/chartjs-plugin-zoom#853] [General]
Hello @kurkle,

I am relying on this plugin in my project and I would like to contribute with some ideas and changes to it. However, I see that the last pull request merged in this repository was in May 4, 2023. I also see a [fork](https://github.com/trullock/chartjs-plugin-zoom), which says "maintenance on the main repos seems to be dead".

Is the maintenance of this repository indeed alive and will issues and PRs be reviewed? In such case, who is the responsible maintainer of this repository?

**Comment by kurkle** [comment]
There is nobody "responsible", because this is open source. I do what and when I can, and so do the other maintainers.

What kind of changes do you have in mind?

**Reply by marisst** [reply]
Thank you for your response. I see that activity has resumed in this repository. For the changes I have in mind, I will open issues etc. when/if I get time.
